flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/dynamic/TestRowDataConverter.java

```diff
@@ -21,7 +21,7 @@
 import static org.apache.iceberg.types.Types.NestedField.optional;
 import static org.apache.iceberg.types.Types.NestedField.required;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
```

Comment:
`BanJUnit5Assertions` rule doesn't fail if the method is imported statically? 

https://github.com/apache/iceberg/blob/main/.baseline/checkstyle/checkstyle.xml

Reply:
Thank you for reviewing the code! I tested it locally and found that the BanJUnit5Assertions rule doesn't work properly when using `import static org.junit.jupiter.api.Assertions.assertThrows`. Do we need to add a new rule to prevent importing methods from `import static org.junit.jupiter.api.Assertions.*`?

Reply:
yes, please add something to checkstyle so that we detect this in the future

Reply:
@nastra @ebyhr Thank you for reviewing the code! I have added new rules to more accurately match specific methods under `org.junit.jupiter.api.Assertions (e.g., assertEquals, assertTrue, etc.)`. With these new rules in place, users will receive prompts if they include these methods.